include/seastar/core/sstring.hh

```diff
@@ -296,6 +294,23 @@ public:
         }
     }
 
+    constexpr size_t find(const char_type* s, size_t pos = 0) const noexcept {
+        return find(s, pos, traits_type::length(s));
+    }
+
+    size_t find(const basic_sstring& s, size_t pos = 0) const noexcept {
+        return find(s.str(), pos, s.size());
+    }
+
+    template<class StringViewLike,
+             std::enable_if_t<std::is_convertible_v<StringViewLike,
```

Comment:
Sanity check: usage of SEASTAR_CONCEPT is now not mandatory, isn't it?

Reply:
Yeah, I think these helper are also handy to C++17 users, so chose to avoid using C++20.